### ISSUE TITLE
Adding files at runtime to the app directory makes watch kick in and rebuild the app

### ISSUE THREAD

**Issue by lambourn** [NativeScript/nativescript-dev-webpack#499]
## Issue Checklist

- [x] You are submitting: a bug report
- [x] You verified that this is a real problem by searching the [NativeScript Forum] and the other [issues] in this repo.
- [x] You checked our [demo apps] and the [documentation] for sample usage.

## Tell us about the problem
When I run `tns run ios --bundle --env.aot` and after app got build and started on physical iOS device, I notice that watch starts a new webpack bundle run even though I did not changed any source files:
```
15:19:49 - Compilation complete. Watching for file changes.

Executing before-watch hook from /Users/cdds/MR/hooks/before-watch/nativescript-dev-webpack.js
Running webpack for iOS...
clean-webpack-plugin: /Users/cdds/MR/platforms/ios/control/app/**/* has been removed.
File change detected. Starting incremental webpack compilation...

Webpack is watching the files…
```
after some debugging I _believe_ (though not 100% sure) that this is due to the fact that the app downloads a file via https to the app's documents directory. 

Is this directory being watched by the liev sync watch as well? And if so can we have some ignore patterns for non source code files?

## Local environment

- [x] Mac 10.13.4
- [x] Node 8.9.1
- [x] NativeScript CLI version 4.0.0

## Project data

- [x] Platform you are building the app for: both (issue is seen on iOS builds)
- [x] Webpack configuration: standard webpack config (updated to latest config after migrating to NS 4.0)

**Comment by lambourn** [comment]
Is there any way to find out which file activity caused the watcher to kick off a new webpack build?

**Comment by tsonevn** [comment]
Hi @lambourn,
This issue seems to be resolved within latest NativeScirpt version 4.1. Try to upgrade to the newest version and check if the problem still will be reproduced. For further help, you can check our [upgrade instructions](https://docs.nativescript.org/releases/upgrade-instructions).
